config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get "password_resets/new"
+  get "password_resets/edit"
```

Comment:
Do you need the routing in line 3 and 2. If so then why?

Reply:
I don't need the routing in line 3 and 2, because they are included in the _resource :password_resets_
I tested at first, I forgot to delete them